pb_node_images_compute.yml

```diff
@@ -24,7 +24,9 @@
 ---
 - name: Node Images Compute
   hosts: packer
-  become: true # Enable builds that do not login as root to escalate privilege.
+  become: yes
```

Comment:
I'd put this back to `true` to appease the linter, and to selfishly make my rebase for #121 easier.

Reply:
Done